meetings/__init__.py

```diff
@@ -173,6 +205,13 @@ async def log_message(self, evt: MessageEvent, match: Tuple[str]) -> None:
       if re.search("\^info", evt.content.body):
         await self.log_tag("info", evt)
         await evt.react("✏️️")
+      
+      # Change the topic
+      if re.search("\^topic", evt.content.body):
+        topic = evt.content.body.removeprefix("^topic").strip()
```

Comment:
I'm wondering longer term if topic-changes should be PL controlled, just like start/end?